icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/UtilityTest.java

```diff
@@ -52,11 +52,9 @@ public void TestUnescape() {
 
         // Regression test for ICU-21645
         String s = "\\U0001DA8B\\U0001DF00-\\U0001DF1E";
-        int[] offset16 = new int[] { 1 };  // after the backslash
         // This returned U+B2F00 for the first _two_ escapes.
-        int c = Utility.unescapeAt(s, offset16);
-        assertEquals(s + " unescape at 1, code point", 0x1DA8B, c);
-        assertEquals(s + " unescape at 1, offset", 10, offset16[0]);
+        int cpAndLength = Utility.unescapeAndLengthAt(s, 1);  // index 1 = after the backslash
+        assertEquals(s + " unescape at 1, cpAndLength", 0x1DA8B09, cpAndLength);
```

Comment:
Should there be a test for unescapeAt? i.e. does this remove all tests for deprecated `unescapeAt()`?

Reply:
This change does replace all uses of the old unescapeAt() in the ICU repo, including in tests.
Would you like me to bring one test back?